client/core/locale_ntfn.go

```diff
@@ -87,6 +87,11 @@ var originLocale = map[Topic]*translation{
 		subject:  "Send Successful",
 		template: "Sending %s %s to %s has completed successfully. Coin ID = %s",
 	},
+	//[order ID, error]
+	TopicAsyncOrderFailure: {
+		subject:  "Inflight Order Error",
+		template: "Inflight order with ID %v failed: %v",
```

Comment:
For user-facing messages, it should be "in-flight".  I don't believe "inflight" is proper English, at best an idiom.

https://dictionary.cambridge.org/dictionary/english/in-flight

Reply:
Although now that I say that, I suspect "in-flight" will prove to be too colloquial to translate directly while keeping the intended meaning.  We can keep "in-flight", but I bet for other languages we'll want translators to give the equivalent of "Submitting Order Error".  Just a thought.

This PR is g2g with the two comments resolved.

Reply:
You really didn't need to change anything in the code, just the *user-facing* text, but it's fine.

I'm unresolving this conversation so that it has visibility when we decide to look back during translation.

Reply:
Alright.